README.md

```diff
@@ -103,15 +133,15 @@ Enables basic full-text search.
 
 ```rb
 # Create the EQL payload using helper functions
-payload = eqlPayload("users", "encrpyted_field", "plaintext value")
+payload = EQL.for_match("users", "encrypted_field", "plaintext value")
```

Comment:
There are probably more idiomatic ways to do this in Ruby but not something we need to worry about now. Just sharing thoughts while they're fresh.

You could do:
```rb
EQL.table("users").column("encrypted_field").match("plaintext value")
```

If you made `EQL.table("users")` a class method:
```rb
class User
  def self.eql
    EQL.table("users")
  end
end
```
And used `method_missing` to handle the column name, you could do something like:
```rb
User.eql.encrypted_field = "foo"
User.eql.encrypted_field > 10
# etc
```
Just ideas for later.

Reply:
Very fair. At the moment I'm mostly aiming for examples that get the point across and would be easy to port to other languages. Agreed that it'll make sense to implement a more idiomatic/ergonomic interface for Ruby at some point.

Reply:
Worth just writting all examples in psuedo code?